test/signalrclient-e2e-tests/hub_connection_tests.cpp

```diff
@@ -181,4 +181,127 @@ TEST(hub_connection_tests, send_message_after_reconnect)
     ASSERT_FALSE(received_event->wait(200));
 
     ASSERT_EQ(*message, U("[\"Send: test\"]"));
-}
\ No newline at end of file
+}
+
+TEST(hub_connection_tests, send_message_empty_param)
+{
+    auto hub_conn = std::make_shared<signalr::hub_connection>(url + U("SignalR"));
+    auto message = std::make_shared<utility::string_t>();
+    auto received_event = std::make_shared<pplx::event>();
+
+    auto hub_proxy = hub_conn->create_hub_proxy(U("hubConnection"));
+
+    hub_proxy.on(U("send"), [message, received_event](const web::json::value& arguments)
+    {
+        *message = arguments.serialize();
+        received_event->set();
+    });
+
+    hub_conn->start().then([&hub_proxy]()
+    {
+        return hub_proxy.invoke<web::json::value>(U("send"));
+
+    }).get();
+
+    ASSERT_FALSE(received_event->wait(200));
```

Comment:
I changed all the tests to wait up to 2000 ms as we spoke before. Please use 2000 in all new commits.